src/main/java/com/ecobike/model/AbstractElectroBike.java

```diff
@@ -0,0 +1,81 @@
+package com.ecobike.model;
+
+import java.util.Objects;
+
+/**
+ * Common class describes electro bike.
+ */
+public abstract class AbstractElectroBike extends Bike {
+
+    /**
+     * Max speed in km/h
+     */
+    private final int maxSpeed;
+
+    /**
+     * Battery capacity in mAh
+     */
+    private final int batteryCapacity;
+
+    public AbstractElectroBike(String brand,
```

Comment:
```suggestion
    public AbstractElectroBike(String brand, int maxSpeed, ...
```
Старайся зайвий раз не розтягувати код

Reply:
done